README.md

```diff
@@ -4,15 +4,70 @@
 
 [![Chi-chan bot Container](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml/badge.svg?branch=master&event=push)](https://github.com/seker212/Discord_bot/actions/workflows/Chi-chan_Container.yml)
 
-## Running locally
+## Table of contents
 
-Clone this repo with checkout to branch `master-2.0`. Run the `DiscordBot` project. :bangbang: Before running remember to add to the main direcotry file `token.txt` with discord bot token.
+1. [Running bot](#tc1)
+    1. [Important informations](#tc11)
+    2. [Local run](#tc12)
+    3. [Doker run](#tc13)
```

Comment:
Let's start with docker, as this is the default way to run the project for non-development purposes. Since we don't provide compiled files, local run is basically just for debug and development